### ISSUE TITLE
[MU4 Issue] [Accessibility] NVDA doesn't read anything (Win)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9155]
**Describe the bug**
NVDA doesn't read anything

**Expected behavior**

**Desktop (please complete the following information):**
 - Win10

**Additional context**
You can download NVDA here www.nvaccess.org
